phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -17,14 +17,20 @@ fi
 mkdir -p $KUBEADM_DIR
 cat <<EOF |tee $KUBEADM_CONFIG_FILE
 kind: MasterConfiguration
-apiVersion: kubeadm.k8s.io/v1alpha1
+apiVersion: kubeadm.k8s.io/v1alpha2
```

Comment:
hmm, I guess we will need to use an old kubernetes-anywhere on the older kubernetes release branches..
Can we make this conditional on the kubeadm or kubernetes veresion?